pkg/querier/querier.go

```diff
@@ -72,3 +72,18 @@ func GetSelfNode(isAgent bool, node string) v1.ObjectReference {
 func GetVersion() string {
 	return version.GetFullVersion()
 }
+
+// NetworkPolicyQueryFilter is used to filter the result while retrieve network policy
+// Not settled attributes means match all.
+// e.g SourceType = "" means all type network policy will be retrieved
+// Can have more attributes in future if more args are required
+type NetworkPolicyQueryFilter struct {
+	// Name of the network policy.
+	Name string `protobuf:"bytes,1,opt,name=name"`
```

Comment:
NetworkPolicyQueryFilter is constructed in-memory and never serialized, the protobuf tag is not needed.

Reply:
Thx!